tests/integration/coldfront_exclusion_test.go

```
@@ -0,0 +1,119 @@
+	_, err = pgCluster.Node2Pool.Exec(ctx, fmt.Sprintf(
+		`INSERT INTO %s (id, val) VALUES (1, 'from_n2') ON CONFLICT DO NOTHING`, qualifiedColdfront))
+	require.NoError(t, err)
+
+	for _, pool := range pools {
+		_, err := pool.Exec(ctx,
+			fmt.Sprintf(`SELECT spock.repset_add_table('%s', '%s');`, repsetName, qualifiedColdfront))
+		require.NoError(t, err)
+	}
+	t.Cleanup(func() {
+		for _, pool := range pools {
+			pool.Exec(ctx, fmt.Sprintf(
```

golangci-lint (2.12.2)

[error] Error return value of `pool.Exec` is not checked

(errcheck)